.github/workflows/flutter_package.yml

```
@@ -23,6 +23,10 @@ on:
         required: false
         type: number
         default: 100
+      optimization:
```

```suggestion
      test_optimization:
```

nit but might be good to keep it specific in case there are other optimizations in the future